api/build.gradle

```diff
@@ -16,9 +16,14 @@ plugins {
 }
 
 apply plugin: 'java'
-apply plugin: 'groovy'
 apply plugin: 'io.spring.dependency-management'
 
+configurations.all {
+    resolutionStrategy {
+        cacheDynamicVersionsFor 0, 'seconds'
```

Comment:
this means gradle will always pick up the latest version for 'latest.release'

Reply:
Nice, good to know!